File: 236/A - Boy or Girl/solution.py
# Read the username from standard input
username = input()
 
# Calculate the number of distinct characters
distinct_chars = set(username)
 
# Determine the gender based on the parity of the distinct count
if len(distinct_chars) % 2 == 0:
    print("CHAT WITH HER!")
else:
    print("IGNORE HIM!")

File: 236/A - Boy or Girl/solution.java
import java.util.Scanner;
 
public class Main {
    public static void main(String[] args) {
        // Create a scanner object to read input from the console
        Scanner scanner = new Scanner(System.in);
        
        if (scanner.hasNext()) {
            String username = scanner.next();
            
            // Count the distinct characters in the username
            long distinctCount = username.chars().distinct().count();
            
            // Check if the number of distinct characters is even or odd
            if (distinctCount % 2 == 0) {
                System.out.println("CHAT WITH HER!");
            } else {
                System.out.println("IGNORE HIM!");
            }
        }
        
        scanner.close();
    }
}